app/src/test/kotlin/edu/stanford/bdh/engagehf/health/symptoms/SymptomsViewModelTest.kt

```diff
@@ -0,0 +1,91 @@
+package edu.stanford.bdh.engagehf.health.symptoms
+
+import com.google.common.truth.Truth.assertThat
+import com.google.firebase.Timestamp
+import edu.stanford.bdh.engagehf.health.HealthRepository
+import edu.stanford.spezi.core.testing.CoroutineTestRule
+import edu.stanford.spezi.core.testing.runTestUnconfined
+import edu.stanford.spezi.core.utils.LocaleProvider
+import io.mockk.coEvery
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.flow.flowOf
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import java.util.Locale
+
+class SymptomsViewModelTest {
+
+    @get:Rule
+    val coroutineTestRule = CoroutineTestRule()
+
+    private val localeProvider: LocaleProvider = mockk()
+    private val healthRepository: HealthRepository = mockk()
+    private val symptomsUiStateMapper: SymptomsUiStateMapper = SymptomsUiStateMapper(localeProvider)
```

Comment:
This should be a mock. And we should stub this based on the public api that we use from the mapper in the view model. The mapper has its own tests, and we don't want to also have failing tests here in case we change some implementation details in the mapper. Also, we might need to add new dependencies the mapper, and we want to minimize the effort of having to adjust this class as well.

And the tests become somehow simplier

```kotlin
val symptoms: List<SymptomScore> = mockk()
coEvery { healthRepository.observeSymptoms() } returns flowOf(Result.success(symptoms))
val uiState: SymptomsUiState = mockk()
every { symptomsUiStateMapper.mapSymptomsUiState(SymptomType.OVERALL, symptoms) } returns uiState
assertThat(viewModel.uiState.value).isEqualTo(uiState)
```